fedimint-rocksdb/src/lib.rs

```diff
@@ -133,6 +135,26 @@ impl<'a> IDatabaseTransaction<'a> for RocksDbTransaction<'a> {
         }))
     }
 
+    async fn raw_remove_by_prefix(&mut self, key_prefix: &[u8]) -> anyhow::Result<()> {
+        fedimint_core::task::block_in_place(|| {
```

Comment:
@m1sterc001guy Notably, I was forced to implement it.

Reply:
We were relying on the default implementation before right?

Reply:
This looks good to me, as long as `verify_remove_by_prefix` is passing I think we're good

Reply:
Yes, but that default impl was meh (loading everything into memory at once), and now it doesn't work due to object-safety anyway.